hostports/watcher.go

```diff
@@ -324,3 +366,57 @@ func setupKernelParameters() error {
 	}
 	return nil
 }
+
+func getBridgeInfo(network metadata.Network, host metadata.Host) (bridge string, bridgeSubnet string) {
+	conf, _ := network.Metadata["cniConfig"].(map[string]interface{})
+	for _, file := range conf {
+		props, _ := file.(map[string]interface{})
+		cniType, _ := props["type"].(string)
+		checkBridge, _ := props["bridge"].(string)
+		bridgeSubnet, _ = props["bridgeSubnet"].(string)
+
+		hostSubnet, _ := host.Labels[hostSubnetLabel]
+		if hostSubnet != "" {
+			bridgeSubnet = hostSubnet
+		}
+
+		if cniType == "rancher-bridge" && checkBridge != "" {
```

Comment:
start with `bridge = dockerBridge`

```
if checkBridge != "" {
    bridge = checkBridge
}
```

Why do we need the check `cniType == "rancher-bridge"`?